### PR TITLE
[cli] Change Ledger defaults

### PR DESCRIPTION
**Pull request by gastonponti**
## Description

Change defaults for `cli` commands with the ledger
--useLedger will retrieve only one address (if the user requires more, will have to use the other known flags)

Add a new flag `--ledgerConfirmAddress`
When is set, before verifying the transaction, the Ledger will require to verify the sending address
Default: never

### CODE REVIEW

File: packages/docs/operations-manual/using-a-ledger-wallet.md [hunk 1]
```diff
@@ -153,6 +153,9 @@ To use `celocli` with your Ledger, ensure the device is connected to your comput
 
 Then, simply append the `--useLedger` flag to any `celocli` commands with which you'd like to use a Ledger.
 
+There is another flag that you can use `--ledgerConfirmAddress`. This flag, allows you to verify from the Ledger the address
```
Comment (codyborn) [inline packages/docs/operations-manual/using-a-ledger-wallet.md]: nitpick: no comma needed after "This flag".

**Review by jmrossy — APPROVED** [review]

File: packages/docs/operations-manual/using-a-ledger-wallet.md [hunk 2]
```diff
@@ -166,3 +169,12 @@ celocli account:authorize --from $ACCOUNT_ADDRESS --role vote --signer $VOTE_SIG
 # If instead you wish to authorize a vote signing key for your ReleaseGold account, plug in the Ledger containing the beneficiary key for the ReleaseGold contract and run the following command.
 celocli release-gold:authorize --contract $RELEASE_GOLD_CONTRACT_ADDRESS --role vote --signer $VOTE_SIGNER_ADDRESS --signature $PROOF_OF_POSSESSION --useLedger
 ```
+
+## Troubleshooting
+
+If you have issues connecting to the Ledger, try the following:
+
+* Check that the Ledger device is connected, powered on, and that you've unlocked it using the PIN.
+* Check that no other applications is using the device (close Ledger Live, or a local Celo Blockchain node)
```
Comment (codyborn) [inline packages/docs/operations-manual/using-a-ledger-wallet.md]: nitpick: "no other applications are"

**Review by codyborn — APPROVED** [review]
LGTM

**Review by clrblmt — COMMENTED** [review]
LGTM

**Review by clrblmt — APPROVED** [review]

**Pull request merged**